### PR TITLE
Added update poll end time & cancel poll operations

### PR DESCRIPTION
**Pull request by khramoff**

### CODE REVIEW

File: Stellar-operation-manage-poll.x [hunk 1]
```diff
@@ -89,7 +89,13 @@ enum ManagePollResultCode
     //: Not allowed to close poll which
     POLL_NOT_READY = -2,
     //: Only result provider is allowed to close poll
-    NOT_AUTHORIZED_TO_CLOSE_POLL = -3
+    NOT_AUTHORIZED_TO_CLOSE_POLL = -3,
+    //: End time is in the past
+    INVALID_END_TIME = -4,
+    //:Only poll owner and admin can cancel poll
```
Comment (artemskriabin) [inline Stellar-operation-manage-poll.x]: As I know space after `//:` reuired

File: Stellar-operation-manage-poll.x [hunk 2]
```diff
@@ -89,7 +89,13 @@ enum ManagePollResultCode
...
+    NOT_AUTHORIZED_TO_CANCEL_POLL = -5,
+    //:Only poll owner can update end time
```
Comment (artemskriabin) [inline Stellar-operation-manage-poll.x]: or admin

Comment (artemskriabin) [reply Stellar-operation-manage-poll.x]: use one error code for both cases (not auth for cancel and update) - just `NOT_AUTHORIZED`
client knows which action he tries to apply

**Review by artemskriabin — DISMISSED** [review]

File: docs/paths/update_poll_end_time.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/updatePollEndTime:
+  get:
+    tags:
+      - Polls
+    summary: Update Poll end time
+    description: |
+                  Allows update poll end time. Only admin or poll owner can cancel poll.
```
Comment (dhaidashenko) [inline docs/paths/update_poll_end_time.yaml]: Cancel?

**Review by dhaidashenko — CHANGES_REQUESTED** [review]

**Review by dhaidashenko — DISMISSED** [review]

**Review by dhaidashenko — APPROVED** [review]

**Pull request merged**